modules/setting/user.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import (
+	"strings"
+
+	"code.gitea.io/gitea/modules/container"
+)
+
+const (
+	UserDeletionKey = "deletion"
+)
+
+// userSetting represents user settings
+type userSetting struct {
+	content container.Set[string]
+}
+
+func (s *userSetting) Enabled(module string) bool {
+	return !s.content.Contains(strings.ToLower(module))
```

Comment:
IMO `strings.ToLower` doesn't make sense. It shouldn't tolerate case-insensitive everywhere.